src/stores/quizStore.ts

```diff
@@ -20,11 +20,11 @@ import { GameData } from '../types';
 
 export const getQuizStore = (set: Function, get: Function) => ({
   getQuizzes: async () => {
-    try {
-      const response = await getQuizzes(isGuestUser() ? -1 : get().userData.userId);
+    if (isGuestUser()) {
+      const response = await getQuizzes(-1);
```

Comment:
Do we have an user in our db with -1 userId?
How does this work anyway?

Reply:
`getQuizzes` fetch data from local storage, for guest user using -1 userId. if not guest user then  fetching data from db